### PR TITLE
FQN checker plugin update.

### PR DESCRIPTION
**Pull request by alexander-aleksandrov**
Javadoc checker plugin obtains 2 parameters.
You need to specify them under the `checkJavadoc` section in a build file that applying plugin.
- "threshold" - a number of wrong links found to make any actions. 
- "responseType" - it can be either "error" or "warn" 
  - "warn" type allows to see all errors in build result records without failing it. 
  - "error" will fails the build process. 
 
```
checkJavadoc {
threshold = <value>
responseType = <value>
}
```

### CODE REVIEW

**Comment by alexander-aleksandrov** [comment]
@dmdashenkov PTAL

File: build.gradle [hunk 1]
```diff
@@ -117,7 +117,18 @@ subprojects {
 
     apply plugin: 'com.google.protobuf'
     apply plugin: 'java'
+
     apply plugin: 'org.spine3.tools.javadoc-checker'
+    // Javadoc checker plugin allows finding FQN links with the wrong format.
+    //"threshold" - is a number of wrong links that should be found to trigger a response.
```
Comment (dmdashenkov) [inline build.gradle]: Please allow a space after `//`

File: build.gradle [hunk 2]
```diff
@@ -117,7 +117,18 @@ subprojects {
...
+    //"responseType" - can be either "error" or "warn"
+    //      "warn" type allows to see all errors in build result records without failing it.
+    //      "error" type will fail the build process.
+    //      For more details please see org.spine3.tools.javadoc-checker.Response
+    checkJavadoc {
+        threshold = 0
```
Comment (dmdashenkov) [inline build.gradle]: According to your description, we need `0` Javadoc style violations to trigger the response.
As I understand this, this is a number of "bearable" violations — the ones we __can__ live with. If it is so, please revisit the description comment.

File: build.gradle [hunk 3]
```diff
@@ -269,3 +280,5 @@ idea.project.ipr {
 apply from: "${rootDir}/scripts/jacoco.gradle"
 
 apply from: publishPlugin
+
```
Comment (dmdashenkov) [inline build.gradle]: Please avoid extra empty lines. One at the end of each file (as the `EOF` symbol) is fairly enough.

File: build.gradle (hunk 1 above)
Comment (alexander-aleksandrov) [inline build.gradle]: Done

File: build.gradle (hunk 2 above)
Comment (alexander-aleksandrov) [inline build.gradle]: Done

File: build.gradle (hunk 3 above)
Comment (alexander-aleksandrov) [inline build.gradle]: Done

**Comment by alexander-aleksandrov** [comment]
@dmdashenkov PTAL

**Pull request merged**